environment/stats/src/cloud/benchflow/collectors/stats.go

```diff
@@ -154,6 +154,21 @@ func stopCollecting(w http.ResponseWriter, r *http.Request) {
 	close(stopChannel)
 	waitGroup.Wait()
 	collecting = false
+	composedMinioKey := ""
+	for _, container := range containers {
+		//minio.GzipFile("./"+container.ID+"_tmp")
+		minio.GzipFile("/app/"+container.ID+"_tmp")
+		minioKey := minio.GenerateKey(container.ID+"_stats.gz")
```

Comment:
This file format used here should be clarified in one of the issues we have open. @Cerfoglg please reply to this comment with the issue where we do that.